newsroom/formatters.py

```diff
@@ -29,6 +29,7 @@ class BaseFormatter:
     FILE_EXTENSION = None
     MEDIATYPE = "text"
     MULTI = False
+    MULTI_ZIP = False
```

Comment:
I couldn't see a use for "MULTI" and was tempted to use it instead, but was afraid I'd missed something so added a new one!

Reply:
You're right, remnants of old code. I have removed it in favour of `MULTI_ZIP` (as it's more descriptive anyway)